app/html/settings/friend-hops.js

```diff
@@ -0,0 +1,84 @@
+const nest = require('depnest')
+const { h, computed, onceTrue, Value, watch } = require('mutant')
+const merge = require('lodash/merge')
+
+exports.gives = nest({
+  'app.html.settings': true
+})
+
+exports.needs = nest({
+  'app.html.settings': 'map',
+  'config.sync.get': 'first',
+  'config.sync.getCustom': 'first',
+  'config.sync.setCustom': 'first'
+})
+
+const friendsHops = 2
+exports.create = function (api) {
+  return nest({
+    'app.html.settings': friendHopsConnections
+  })
+
+  function friendHopsConnections () {
+    let hops = Value(api.config.sync.get('friends.hops', friendsHops))
+    hops.initial = hops()
+
+    watch(hops, hops => {
+      const intHops = parseInt(hops)
+      updateConfig(intHops)
+    })
+
+    return {
+      title: 'Friend Hops',
+      body: h('Friends', [
```

Comment:
make this `h('FriendsHops', ...)`

This is shorthand for `h('div.FriendsHops', ...)  which makes a div with class `.FriendsHops`. If you name it this it will match the MCSS class you made and things should line up !

Reply:
oh right, thank you!